mParticle/mparticle-unity-plugin
# Add ability to set environment + other small fixes

### Summary of changes:
- Adds ability to set environment in the initialize function
- iOS framework fixed
- getEnvironment in Android fixed
- Post processor file moved to Editor folder (works with latest version of unity)
- Updated README instructions for iOS

### Test
- On Mac, run the build.sh script (or just copy the command and run it directly)
- Create an Android project, set the environment and then do get environment to make sure it is set correctly.
- Repeat for iOS (see README for more detailed build instructions).
